api/src/main/java/com/community/api/services/DocumentStorageService.java

```diff
@@ -203,25 +242,25 @@ public void saveAllDocumentTypes() {
-
-                new DocumentType(24,"PERSONAL_PHOTO", "A Personal Photgraph of SP"),
-
-
-                new DocumentType(25, "CATEGORY", "The classification of individuals, such as gender categories: Male, Female, Other."),
-                new DocumentType(26, "DISABILITY", "A term used to describe individuals with physical or mental impairments; 'person with a disability' is the preferred terminology."),
-                new DocumentType(27, "EX-SERVICE-MEN", "An identification document required for veterans, typically used to access benefits or services."),
-                new DocumentType(28, "NCC", "A document serving as proof of participation in the National Cadet Corps, often required for certain government applications."),
-                new DocumentType(29, "SPORTS", "A personal photograph typically required for sports-related documentation, such as player registrations or team memberships."),
-                new DocumentType(30, "FREEDOM FIGHTER", "A personal photograph required for identification and documentation purposes related to recognition and benefits for freedom fighters.")
+//                new DocumentType(25,"NCC CERTIFICATE A", "NCC CERTIFICATE A"),
+//                new DocumentType(26,"NCC CERTIFICATE B", "NCC CERTIFICATE B"),
+//                new DocumentType(27,"NCC CERTIFICATE C", "NCC CERTIFICATE C"),
+//                new DocumentType(28,"NSS CERTIFICATE", "NSS CERTIFICATE"),
+//                new DocumentType(29,"SPORTS CERTIFICATE - STATE", "SPORTS CERTIFICATE FOR STATE LEVEL"),
+//                new DocumentType(30,"SPORTS CERTIFICATE - CENTRE", "SPORTS CERTIFICATE FOR STATE LEVEL"),
+//                new DocumentType(23,"BUSSINESS_PHOTO", "A Standard proof of Running Bussiness"),
+//
+//                new DocumentType(24,"PERSONAL_PHOTO", "A Personal Photgraph of SP"),
+//
+//                new DocumentType(24,"PERSONAL_PHOTO", "A Personal Photgraph of SP"),
+//
+//
+//                new DocumentType(25, "CATEGORY", "The classification of individuals, such as gender categories: Male, Female, Other."),
+//                new DocumentType(26, "DISABILITY", "A term used to describe individuals with physical or mental impairments; 'person with a disability' is the preferred terminology."),
+//                new DocumentType(27, "EX-SERVICE-MEN", "An identification document required for veterans, typically used to access benefits or services."),
+//                new DocumentType(28, "NCC", "A document serving as proof of participation in the National Cadet Corps, often required for certain government applications."),
+//                new DocumentType(29, "SPORTS", "A personal photograph typically required for sports-related documentation, such as player registrations or team memberships."),
+//                new DocumentType(30, "FREEDOM FIGHTER", "A personal photograph required for identification and documentation purposes related to recognition and benefits for freedom fighters.")
 
```

Comment:
use multiline comments and while generating a PR try to remove commented code (can also add commented code as well but only neccesary).